kf/nfconfig.go

```diff
@@ -937,3 +938,219 @@ func getHostInterfaces() (map[string]bool, error) {
 	}
 	return hostIfaces, nil
 }
+
+//  AddProgram : AddProgram will add given programs on given interface
+func (c *NFConfigs) AddProgram(ifaceName, hostName string, bpfProgs *models.BPFPrograms) error {
```

Comment:
if the comment in 942 is correct (that multiple can be done in the same call) then this should be renamed
```suggestion
//  AddPrograms : AddPrograms will add given programs on given interface
func (c *NFConfigs) AddPrograms(ifaceName, hostName string, bpfProgs *models.BPFPrograms) error {
```